routes/route.js

```diff
@@ -0,0 +1,28 @@
+exports.get = {

```

Comment:
can we move this into the test folder?  It's a little confusing on the top level